### DISCUSSION TITLE
"Failed to compute cache key" with Docker build (indie stack)

### DISCUSSION THREAD

**Discussion by nicolasrenon** [remix-run/remix#3586] [Q&A] [answered]
Hello everyone.

I'm stuck with this error from Docker in the Github Actions:

```bash
#11 [production-deps 3/4] ADD package.json package-lock.json ./
#11 ERROR: failed to calculate checksum of ref tjkd8n4jjuntlq4pxebuohut5::nclwdmh6iwewjn9irs0gnde8m: "/package-lock.json": not found
```

```bash
WARNING: local cache import at /tmp/.buildx-cache not found due to err: could not read /tmp/.buildx-cache/index.json: open /tmp/.buildx-cache/index.json: no such file or directory
Dockerfile:15
--------------------
  13 |     WORKDIR /myapp
  14 |     
  15 | >>> ADD package.json package-lock.json ./
  16 |     RUN npm install --production=false
  17 |     
--------------------
error: failed to solve: failed to compute cache key: failed to calculate checksum of ref tjkd8n4jjuntlq4pxebuohut5::nclwdmh6iwewjn9irs0gnde8m: "/package-lock.json": not found
Error: buildx failed with: error: failed to solve: failed to compute cache key: failed to calculate checksum of ref tjkd8n4jjuntlq4pxebuohut5::nclwdmh6iwewjn9irs0gnde8m: "/package-lock.json": not found
```

I'm using the Indie Stack.

A quick [Google search](https://www.google.com/search?q=docker+failed+to+compute+cache+key) gives me pretty much the same explanation like

>The Docker failed to compute cache key error is result of docker build command which is executed on a directory where the Dockefile is not present.

I haven't changed much in my codebase from the template, so I'm not sure how I ended up with this error...

**Comment by nicolasrenon** [comment, marked as answer]
Nevermind, I didn't notice before but it seems to be [currently](https://github.com/remix-run/indie-stack/actions?query=branch%3Amain) the case for the main branch of remix-run/indie-stack.
The last successful deploy was [2 months ago](https://github.com/remix-run/indie-stack/actions/runs/2260135666).

**Comment by muco-rolle** [comment]
What is the solution for @nicolasrenon
